docs/supported_backups.md

```diff
@@ -0,0 +1,31 @@
+# OADP Supported Backups
+
+OADP supports backup and restore disaster recovery operations for both containerized and OpenShift Virtualization workloads. It is important to distinguish the nuances and support for each workload type.
+
+## Containerized Workloads
+
+OADP supports backup and restore operations for containerized workloads. 
+
+| VolumeMode | FSB - restic (i) | FSB - kopia | CSI | CSI DataMover (ii) | Cloud Provider Native Snapshots |
+|------------|--------------|-------------|-----|------------------| ----------------|
+| Filesystem | S, I         | S, I        | S   | S, I             | S               |
+| Block      | N            | N           | S   | S, I             | S               |
+
+
+## OpenShift Virtualization Workloads
+
+OADP supports backup and restore operations for OpenShift Virtualization workloads.
+
+| VolumeMode | FSB - restic (i) | FSB - kopia | CSI | CSI DataMover (ii) | Cloud Provider Native Snapshots |
+|------------|--------------|-------------|-----|------------------| ----------------|
+| Filesystem | N            | N           | S   | S, I             | N               |
+| Block      | N            | N           | S   | S, I             | N               |
+
+* Legend:
+  * **S:** Supported
+  * **I:** Incremental Backup supported
+  * **N:** Not Supported
+  * **FSB:** File System Backup
+  * **Cloud Provider Native Snapshots:** Non CSI Volume support for AWS, GCP, Azure via plugins.
```

Comment:
Maybe just update this to: Non CSI Volume support for AWS, GCP, Azure via VolumeSnapshotter plugins.
WDYT @kaovilai ?

Reply:
Sure. Perhaps also have to change description to say

```diff
+OADP supports volume snapshot operations for OpenShift Virtualization workloads.
-OADP supports backup and restore operations for OpenShift Virtualization workloads.
```

Reply:
I don't think `volume snapshot operations` is clear enough for customers.  They are using OADP to backup and resore or disaster recovery.  Not to snapshot volumes.